package.json

```diff
@@ -15,12 +15,13 @@
     "format": "npx prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md)\"",
     "knex": "npx knex --knexfile config/knexfile.js",
     "tests": "npx jest",
-    "coverage": "CI=true npx jest --coverage --detectOpenHandles --forceExit"
+    "coverage": "CI=true npx jest --coverage --detectOpenHandles --forceExit",
+    "prepare": "husky install"
   },
   "lint-staged": {
     "api/**/*.js": [
-      "eslint --fix",
-      "prettier-eslint --write"
+      "npm run lint:fix",
```

Comment:
I haven't used lint, just a general question, what happens when we run this command? Do we get blocked submitting code that hasn't been linted?

Reply:
You should get blocked from send the code up. You can run "npm run lint:fix" to resolve all your linting errors before you commit any code.